### PR TITLE
Registration: change login verbiage

### PR DESCRIPTION
**Pull request by syoungfntech**
https://tipit.avaza.com/project/view/347915#!tab=task-pane&groupby=ProjectSection&view=vertical&task=3408514&stafilter=NotComplete&fileview=grid

### CODE REVIEW

File: resources/js/login/login.js [hunk 1]
```diff
@@ -368,7 +368,7 @@ const ThirdPartyIdentityProviders = ({ thirdPartyProviders, formAction, disableI
                             target="_self"
                             title={`Sign In with ${provider.label}`}
                             href={`${formAction}/${provider.name}`}>
-                            {provider.label}
+                            {provider.label === "facebook" ? "Log in with " : "Sign in with "}{provider.label}
```
Comment (smarcet) [inline resources/js/login/login.js:371]: this is not correct
if u check the source of this page
[SpeakersSummitRegistrationPromoCode](view-source:https://idp.dev.fnopen.com/auth/login)
labels are
 thirdPartyProviders: [
                                {label: "Facebook", name:"facebook"},
                                 {label: "Apple", name:"apple"},
                                 {label: "Linkedin", name:"linkedin"},
                                 {label: "Okta", name:"okta"},
                             ],

Comment (smarcet) [reply resources/js/login/login.js:371]: also lets create a helper method and give it a meaningful name so you could tell the rationale of this just reading the code

**Review by smarcet — CHANGES_REQUESTED** [review]
@syoungfntech please review comments

File: resources/js/login/login.js (hunk 1 above)
Comment (syoungfntech) [reply resources/js/login/login.js]: @smarcet I am going to put the helper method in `resources/js/utils.js` if there's no objection

Comment (smarcet) [reply resources/js/login/login.js]: sure also check that we need to same for title of the anchor element
regards

**Review by romanetar — APPROVED** [review]
LGTM

**Review by smarcet — APPROVED** [review]
LGTM

**Pull request merged**